core/src/components/content/content.tsx

```diff
@@ -75,6 +75,15 @@ export class Content implements ComponentInterface {
    */
   @Prop() fullscreen = false;
 
+  /**
+   * Controls where the fixed content is placed relative to the main content
+   * in the DOM. This can be used to control the order in which fixed elements
+   * receive keyboard focus.
+   * For example, if a FAB in the fixed slot should receive keyboard focus before
+   * the main page content, set this property to `'beforeContent'`.
+   */
+  @Prop() fixedSlotPlacement: 'afterContent' | 'beforeContent' = AFTER_CONTENT_SLOT_POSITION;
```

Comment:
I created const variables to refer to each of these values. We use them in multiple places in Content, so I think it will be easy to mistype something.

The idea here is that if developers can refer to the values by using the consts. If the types change but the const are not updated then devs should get an error on build.